### ISSUE TITLE
Configuring latitude value with a storyboard is broken

### ISSUE THREAD

**Issue by tmcw** [mapbox/mapbox-gl-native#3868]
<img width="590" alt="2016-02-09 at 9 36 am" src="https://cloud.githubusercontent.com/assets/32314/12919126/97140c36-cf10-11e5-944b-74f018cc59dd.png">

It doesn't matter what you put in the latitude field, the map is always at the equator

**Comment by 1ec5** [comment]
We’re setting the right center coordinate initially. Immediately thereafter, when we apply content inset, we have to capture that center coordinate then reapply it after applying the content inset. For some reason, the call to `-centerCoordinate` within `-setContentInset:animated:` is returning a 0 latitude (but the correct longitude).

```
#0	0x000000010e9ecab9 in -[MGLMapView _setCenterCoordinate:edgePadding:zoomLevel:direction:duration:animationTimingFunction:completionHandler:] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:1772
#1	0x000000010e9ec9f6 in -[MGLMapView _setCenterCoordinate:animated:] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:1767
#2	0x000000010e9e1691 in -[MGLMapView setContentInset:animated:] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:862
#3	0x000000010e9e14ff in -[MGLMapView setContentInset:] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:842
#4	0x000000010e9e12f7 in -[MGLMapView adjustContentInset] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:837
#5	0x000000010e9e07ca in -[MGLMapView layoutSubviews] at /path/to/mapbox-gl-native/platform/ios/src/MGLMapView.mm:785
```

**Comment by 1ec5** [comment]
Reproduces if and only if you set the latitude or longitude after the zoom level. If you set the zoom level first, the map view winds up in the correct location.

**Comment by 1ec5** [comment]
Fixed in #3887, which will be in 3.1.1 shortly.
